objc/blocks_runtime.h

```diff
@@ -2,35 +2,28 @@
 #pragma clang system_header
 #endif
 
+
 /*
  * Blocks Runtime
  */
 #include "Availability.h"
 #ifdef __cplusplus
-#include <stdbool.h>
+#define OBJC_BOOL_TYPE bool
+#define OBJC_BLOCK_PTR_TYPE void*
```

Comment:
Is this consistent with Apple?  I'm not sure why we ended up with `void*` in Objective-C++ and `id` in Objective-C.

Reply:
OK.  maybe i'll refactor to be as consistent with apple as possible (they seem to include #include <stdbool.h> and just use bool as return type as well).  That may be best moving forward for compatibility with their header.

Reply:
OK, should look better now.